### ISSUE TITLE
cgi is deprecated and slated for removal in Python 3.13

### ISSUE THREAD

**Issue by cclauss** [webpy/webpy#732]
Pytest on Python 3.11 pre-release will generate...
```
web/webapi.py:6: DeprecationWarning: 'cgi' is deprecated and slated for removal in Python 3.13
    import cgi
```
https://docs.python.org/3/library/cgi.html

**Comment by JakeWalker23** [comment]
Hi @cclauss,

What is needed to be done here? I don't mind doing this. 

Jake

**Comment by cclauss** [comment]
https://peps.python.org/pep-0594/#cgi

**Comment by scottbarnes** [comment]
I wonder if [streaming-form-data](https://pypi.org/project/streaming-form-data/) might work for replacing `cgi.FieldStorage`.

I used it to [parse some form data](https://github.com/internetarchive/openlibrary-bots/pull/258/files#diff-af09e26e8a81aef9a643b499c3fd26c4572157dc8019ca565dade99aae1e1debR46) when mocking a POST with [openlibrary-client](https://github.com/internetarchive/openlibrary-client/). If I mocked that correctly, it's doing the work of `cgi.FieldStorage`. The key word there being *if*.

Any thoughts on this, @cclauss or @JakeWalker23?

**Comment by cclauss** [comment]
I have no expertise in the area of [cgi](https://peps.python.org/pep-0594/#cgi) but I do believe that a pull request would give the community something to throw darts at so I would encourage that.

**Comment by tfmorris** [comment]
I think this was resolved by #773 and can be closed.
